Src/library/base/elks/kernel/type.e

```diff
@@ -1,12 +1,13 @@
 note
 	description: "Representation of an Eiffel type."
+	remark: "At any given time, there is no more than one instance of TYPE representing a given Eiffel type."
 	library: "Free implementation of ELKS library"
 	status: "See notice at end of class."
 	legal: "See notice at end of class."
 	date: "$Date$"
 	revision: "$Revision$"
 
-class
+frozen class
```

Comment:
This seems necessary based on the assumption made in the implementation of the built-in features of this class (see eif_built_in.h) that the current object is of generic type whose first actual generic parameter is supposed to represent the Eiffel type being handled. If we allow inheriting from this class, then a descendant class may have no generic parameter, or one which is not related to the type being handled. This will break the implementation of the inherited built-in features.